src/components/picker/helpers/usePickerSelection.tsx

```diff
@@ -1,15 +1,23 @@
 import {RefObject, useCallback, useState, useEffect} from 'react';
 import _ from 'lodash';
-import {PickerProps, PickerValue, PickerSingleValue, PickerMultiValue, PickerModes} from '../types';
+import {
+  PickerProps,
+  PickerValue,
+  PickerSingleValue,
+  PickerMultiValue,
+  PickerModes,
+  PickerPropsDeprecation
+} from '../types';
 
-interface UsePickerSelectionProps
-  extends Pick<PickerProps, 'migrate' | 'value' | 'onChange' | 'getItemValue' | 'topBarProps' | 'mode'> {
-  pickerExpandableRef: RefObject<any>;
-  setSearchValue: (searchValue: string) => void;
-}
+type UsePickerSelectionProps = Pick<PickerProps, 'value' | 'onChange' | 'mode' | 'onCancel'> &
```

Comment:
I didn't mean to add 'onCancel' prop to picker prop, but to have in usePickerSelection the 'onCancel' prop instead of 'headerProps'. The picker component will pass the hook it from the headerProps